### PR TITLE
chore(test): Add widget test coverage

### PR DESCRIPTION
**Pull request by Pessimistress**
#### Change List
- Tests for widgets module

### CODE REVIEW

File: test/modules/widgets/scale-widget.spec.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+afterEach(() => {
+  testInstance?.destroy();
+  testInstance = undefined;
+});
+
+test('ScaleWidget', async () => {
+  testInstance = new WidgetTester({
+    widgets: [new ScaleWidget()]
+  });
+
```
Comment (chrisgervang) [inline test/modules/widgets/scale-widget.spec.ts:26]: Nothing is being tested in this one. Add a test here to verify the scale calculations given a view state?

File: test/modules/widgets/splitter-widget.spec.ts [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+  const widget = new SplitterWidget<MapView[]>({
+    viewLayout: mapsLayoutH,
+    onChange: newViews => {
+      onViewsChange(newViews);
+      testInstance.setProps({views: newViews});
+    }
+  });
+  testInstance = new WidgetTester<MapView[]>({
+    views: [],
+    initialViewState: {
```
Comment (chrisgervang) [inline test/modules/widgets/splitter-widget.spec.ts:202]: Aren't users expected to use viewState in controlled mode?

**Review by chrisgervang — APPROVED** [review]

File: test/modules/widgets/geocoder-widget.spec.ts [hunk 3]
```diff
@@ -50,7 +66,7 @@ test('CoordinatesGeocoder.geocode - Returns null for invalid inputs', async () =
 test('CoordinatesGeocoder.geocode - Parses mixed formats and boundary conditions', async () => {
   const cases = [
     {input: '85°, -180°', expected: {latitude: 85, longitude: -180}},
-    {input: '85°0\'0\"N 180°0\'0\"E', expected: {latitude: 85, longitude: 180}},
+    {input: '85°0\'0\"N, 180°0\'0\"E', expected: {latitude: 85, longitude: 180}},
```
Comment (chrisgervang) [inline test/modules/widgets/geocoder-widget.spec.ts:70]: Test acceptance of Google Maps context menu copied coords? 

`lat, lng`

File: test/modules/widgets/splitter-widget.spec.ts (hunk 2 above)
Comment (Pessimistress) [reply test/modules/widgets/splitter-widget.spec.ts:202]: only the `views` prop is being controlled here
